pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/PasswordFileCreator/PasswordFileCreatorCommand.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.tools.pravegacli.commands.PasswordFileCreator;
+
+import com.google.common.base.Strings;
+import io.pravega.tools.pravegacli.commands.CommandArgs;
+import lombok.Cleanup;
+import io.pravega.tools.pravegacli.commands.Command;
+import io.pravega.controller.server.rpc.auth.StrongPasswordProcessor;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
+public class PasswordFileCreatorCommand extends Command {
+    static final String COMPONENT = "PasswordFileCreator";
```

Comment:
Please, rename this constant to be consistent with the package name (e.g., `admin`).